EnglishFilter/sections/specific.txt

```diff
@@ -507,6 +507,7 @@ thewpclub.net##.style1 > .note2_sp
 thewpclub.net##.widget_better-ads
 thewpclub.net##.bsac
 webmd.com##div.ad[id^="ads"]
+||sdad.guru/i10c@p1^$domain=webmd.com,third-party
```

Comment:
better try this `||g00.webmd.com^$script`

Reply:
it is hiding too much
![here](https://user-images.githubusercontent.com/45171506/49588934-1d231e80-f979-11e8-8c36-357c77d19c3b.png)

Reply:
Ok. But this is InstartLogic ads reinjection script. Please try JS rules from Eng filter.